src/neuroconv/tools/roiextractors/imagingextractordatachunkiterator.py

```diff
@@ -82,6 +82,17 @@ def __init__(
             progress_bar_options=progress_bar_options,
         )
 
+    def _get_scaled_chunk_shape(self, chunk_mb: float) -> tuple:
+        """Select the chunk_shape less than the threshold of chunk_mb that is also a multiple of the image size."""
+        assert chunk_mb > 0, f"chunk_mb ({chunk_mb}) must be greater than zero!"
+
+        image_shape = self._maxshape[1:]
+        frame_size_bytes = np.prod(image_shape) * self._dtype.itemsize
```

Comment:
Fun fact from a nasty bug in HDMF a while back: always use `math.prod` instead of `np.prod`

Reason: buffer overflow